src/ui/details/hidden_fields.rs

```diff
@@ -49,16 +45,15 @@ pub fn hidden_fields(param: HiddeFields) -> impl View {
 
 	let expand_icon = include_str!("../icons/expand.svg");
 	let contract_icon = include_str!("../icons/contract.svg");
+	let line = include_str!("../icons/line.svg");
```

Comment:
I really couln't stand the line anymore and there are no concepts of dotted or dashed in floem...